backend/danswer/configs/app_configs.py

```diff
@@ -96,6 +96,8 @@
 # 1 edit per 2 characters, currently unused due to fuzzy match being too slow
 QUOTE_ALLOWED_ERROR_PERCENT = 0.05
 QA_TIMEOUT = 10  # 10 seconds
+# Include additional document/chunk metadata in prompt to GenerativeAI
```

Comment:
doesn't work great, need prompt tuning, main problem is that GPT confuses aggregations with metadata... any questions asked on metadata it refuses to answer